apriltag_ros/include/apriltag_ros/common_functions.h

```diff
@@ -171,6 +173,10 @@ class TagDetector
   int max_hamming_distance_ = 2;  // Tunable, but really, 2 is a good choice. Values of >=3
                                   // consume prohibitively large amounts of memory, and otherwise
                                   // you want the largest value possible.
+  double fov_scale_;
```

Comment:
```suggestion
  double fov_buffer_scale_;
```

needs a comment describing what this does

Reply:
Put it in sim_setting.yaml in the other repo but maybe I'll move it here and add a note in settings.yaml saying that it's here (to avoid having it written in multiple places).